demo/build.sh

```diff
@@ -60,7 +60,7 @@ check_port_avaliable()
 {
     port=$1
     name=$2
-    if [ "$(netstat -na 2>/dev/null |grep $port | wc -l)" -ne "0" ]; then
+    if [ "$(netstat -na 2>/dev/null |grep '\<$port\>' | wc -l)" -ne "0" ]; then
```

Comment:
```suggestion
   if [ "$(lsof -i:$port | wc -l)" -ne "0" ]; then
```
可支持mac和linux，参考：    if [ "$(lsof -i:$port | wc -l)" -ne "0" ]; then

Reply:
这条命令在mac上是支持的，我是参照你们原本的写法，只不过原本的是模糊查询，经过测试会经常查出无关的端口，所以改成精确查询就好了